code/modules/clothing/under/miscellaneous.dm

```diff
@@ -1,759 +1,759 @@
-/obj/item/clothing/under/pj/red

```

Comment:
Why did it dequl the file and remake it?
Like this added like 750~ line edits for no reason